project/Build.scala

```diff
@@ -160,6 +161,10 @@ object AlgebirdBuild extends Build {
   lazy val algebirdBijection = module("bijection").settings(
     libraryDependencies += "com.twitter" %% "bijection-core" % "0.8.0"
   ).dependsOn(algebirdCore, algebirdTest % "test->test")
+
+  lazy val algebirdSpark = module("spark").settings(
+    libraryDependencies += "org.apache.spark" %% "spark-core" % "1.3.0" % "provided"
```

Comment:
Its provided so the user brings their own. Once the API's ABI compatible in spark this uses it should work fine for both